lstchain/reco/dl1_to_dl2.py

```diff
@@ -450,8 +450,9 @@ def get_source_dependent_parameters(data, config={}):
 
     src_dep_params = pd.DataFrame(index=data.index)
 
-    is_simu = ('mc_type' in data.columns) and ~(data['mc_type'] < 0).all()
-    
+    is_simu = ~(data['mc_type'] < 0).all() if ('mc_type' in data.columns) else False
```

Comment:
`~(data['mc_type'] < 0).all()`  all is very confusing to me. And I don't even think it is correct. 
Shouldn't this simply be `(data['mc_type'] > 0).all()` ?

Also you don't need parenthese around the if condition.

Reply:
As mentioned above, the current gamma MC has -1 or 0 and for real data has -1 for all events as `mc_type`. 
So something like this?
```
is_real = (data['mc_type'] == -1).all() if 'mc_type' in data.columns else True
is_simu = not is_real
```